addons/main/functions/debug/fnc_debugDraw.sqf

```diff
@@ -119,42 +119,49 @@ private _fnc_debug_drawRect = {
 
 };
 
+private _sideUnknownColor = GVAR(debug_sideColorLUT) get sideUnknown;
 {
     private _unit = _x;
-    private _renderPos = getPosVisual _unit;
-    if (((positionCameraToWorld [0, 0, 0]) distance _renderPos) <= 1000) then {
-    // if (true) then {
+    private _renderPos = getPosATLVisual _unit;
+    if (((positionCameraToWorld [0, 0, 0]) distance _renderPos) <= viewDistance) then {
```

Comment:
Minor improvement to move the call of `viewDistance` out of the loop into some private variable?